proposals/wgs/wg-charter.md

```diff
@@ -0,0 +1,71 @@
+# WG Charter Guide
+
+All Kubeflow WGs must define a charter defining the scope and governance of the WG.
+
+- The scope must define what areas the WG is responsible for directing and maintaining.
+- The governance must outline the responsibilities within the WG as well as the roles
+  owning those responsibilities.
+
+## Steps to create a WG charter
+
+1. Copy [the template][Short Template] into a new file under community/wg-*YOURWG*/charter.md ([sig-architecture example])
```

Comment:
@andreyvelich do you just mean drop the *?

Reply:
The example here is broken: https://github.com/kubeflow/community/pull/348/files#diff-feea637db63d75ae2cdc9794c2d40540R65

Reply:
@jlewi, I think `/charter.md` is unnecessary here, since we are talking about moving file to the directory. 
```suggestion
1. Copy [the template][Short Template] into a new file under community/wg-*YOURWG* ([sig-architecture example])
```